src/vs/platform/opener/common/opener.ts

```diff
@@ -32,7 +32,7 @@ export type OpenInternalOptions = {
 	readonly fromUserGesture?: boolean;
 };
 
-export type OpenExternalOptions = { readonly openExternal?: boolean; readonly allowTunneling?: boolean };
+export type OpenExternalOptions = { readonly openExternal?: boolean; readonly allowTunneling?: boolean, readonly useDefault?: boolean };
```

Comment:
I actually just pushed a fix for https://github.com/microsoft/vscode/issues/114543 that conflicts with the opener part of the change (although I think the logic is very similar)

Basically instead of the `useDefault` option this PR adds, I added an `allowContributedOpeners` option that defaults to false. Where you were passing in `useDefault`, I think you would not set `allowContributedOpeners` for the same behavior